Specs/DateTime/Spanish/TimePeriodExtractor.json

```diff
@@ -219,17 +219,6 @@
       }
     ]
   },
-  {
-    "Input": "Estaré fuera de 4pm a cinco treinta",
```

Comment:
This test seems a correct equivalent to `I'll be out from 4pm to half past five`. Also, we could add `Estaré fuera de 4pm a cinco y treinta`.